src/routes/ProtectedRoute.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Navigate, Outlet } from "react-router-dom";
+import { useAuth } from "../context/auth-context";
+
+export function ProtectedRoute() {
+  const { user } = useAuth();
+  console.log("from protected", user);
```

Comment:
Remove unwanted ```console.log```

Reply:
Yeah, I forgot to remove them. Thanks for pointing out